### ISSUE TITLE
No outgoing packets

### ISSUE THREAD

**Issue by outrunthewolf** [zebrafishlabs/nginx-statsd#13]
My statsd server is not receiving any packets from nginx, and as far as I can tell, the system is not sending any UDP requests.

I've only monitored:

    netstat -npuc

to check for outgoing UDP transactions. Nothing shows up.

I'm running nginx 1.7.9, with spdy, ssl, and this module. Any ideas you have to help me figure out what's going on here?

**Comment by lyrixx** [comment]
What statsd server config did you used ? I had to put the port like that: `statsd_server 127.0.0.1:2003;`

**Comment by outrunthewolf** [comment]
I'll have to give it a try again. Its been a while since I tried first time.

**Comment by lyrixx** [comment]
yes, this package seems not really maintained. But it works for us ;)
